src/features/events/components/EventsList/index.tsx

```diff
@@ -21,15 +27,15 @@ export const EventsList: FC = () => {
 
   const events = useMemo(() => {
     if (filter === FilterType.ALL) {
-      return data.sort((a, b) => (a.startsAt < b.startsAt ? -1 : 1))
+      return [...data].sort(sorts.ascending)
     } else if (filter === FilterType.FUTURE) {
       return data
         .filter((event) => isAfter(new Date(event.startsAt), new Date()))
-        .sort((a, b) => (a.startsAt < b.startsAt ? -1 : 1))
+        .sort(sorts.ascending)
```

Comment:
In this case we don't need to manually create the new array by hand because `filter` actually does that for us! Neat.